### PR TITLE
 #156873054 add office

### PR DESCRIPTION
**Pull request by proxiex**
#### What does this PR do?
- create office model and schema
- add office model and schema
- write test cases for both schema and model
- modify location relationship, test files and fixtures
- modify block relationship, test files and fixtures

#### Description of Task to be completed?
- Add an office to a location
#### How should this be manually tested?
- run app
- visit http://localhost:5000/mrm

#### Any background context you want to provide?
n/a

#### What are the relevant pivotal tracker stories?
[156873054](https://www.pivotaltracker.com/n/projects/2154921)

#### Screenshots (if appropriate)
screen shot 2018-06-26 at 1 31 17 pm

#### Questions:
n/a

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -19,24 +19,18 @@
...
 
->>>>>>> 886bd3f4174947f2fb32aba10b8f635d69cbd92a
 from helpers.database import Base
 from api.location.models import Location
 from api.block.models import Block
 from api.floor.models import Floor
 from api.room.models import Room
 from api.room_resource.models import Resource
-
+from api.office.model import Office
```
Comment (PatrickCmd) [inline alembic/env.py]: I think office is the same as block, why introduce another model **office**

Comment (proxiex) [reply alembic/env.py]: I don't think they are the same thing. An office "EPIC Tower" for example could have block A, B, C etc.

Comment (PatrickCmd) [reply alembic/env.py]: Then the relationship should also change, location is related to office, **blocks = relationship('Block')** in **location model** changes to **offices = relationship('Office') ** and office is related to blocks, **location = relationship('Location')** changes to **blocks = relationship('Blocks')** in **office model**

Comment (proxiex) [reply alembic/env.py]: I guess it should. But will I be the one to do that?

Comment (PatrickCmd) [reply alembic/env.py]: Exactly yes. You are.

Comment (proxiex) [reply alembic/env.py]: 🤨

**Pull request merged**